subsys/net/lib/sockets/Kconfig

```diff
@@ -30,6 +30,15 @@ config NET_SOCKETS_POLL_MAX
 	help
 	  Maximum number of entries supported for poll() call.
 
+config NET_SOCKETS_CONNECT_TIMEOUT
+	int "Timeout value in milliseconds to CONNECT"
+	default 3000
+	range 0 60000
+	help
+	 This variable specifies time in milliseconds after user
```

Comment:
The help text needs some clarifications. Better would be to say something like this:
```
This variable specifies time in milliseconds after connect() API call will timeout
if we have not received SYN-ACK from peer.
```

Reply:
done.